tests/data/vhr10/data.py

```diff
@@ -54,12 +53,7 @@ def generate_test_data(root: str, n_imgs: int = 3) -> str:
             "segmentation": [[1, 1, 2, 2, 3, 3, 4, 5, 5]],
             "iscrowd": 0,
         }
-        if i != 0:
-            ANNOTATION_FILE["annotations"].append(annot)
-        else:
-            noseg_annot = deepcopy(annot)
-            del noseg_annot["segmentation"]
-            ANNOTATION_FILE["annotations"].append(noseg_annot)
+        ANNOTATION_FILE["annotations"].append(annot)
```

Comment:
Did something change in the dataset such that all samples now have segmentations? I didn't think that was the case for the negative set.

Reply:
No, this check was implemented to simulate a case where segmentations were missing from the dataset. However recently I got to check the dataset and the positive set did all have segmentations making this unnecessary. The negative set has no annotations (so no boxes. labels, segmentations etc)

Reply:
So don't we still need this code to simulate the negative set?

Reply:
No. The negative set is already simulated by having a directory of images in a negative test folder. This bit was used to simulate a situation where there was no segmentations in the positive set. Which is not the case in the dataset since all samples in the positive set do have segmentations.